segmentstore/server/src/main/java/io/pravega/segmentstore/server/containers/ContainerEventProcessorImpl.java

```diff
@@ -427,8 +429,8 @@ protected Duration getShutdownTimeout() {
          */
         private void reconcileStartOffset() {
             long newStartOffset = segment.getInfo().getStartOffset();
-            log.info("{}: Reconciling start offset from {} to {}.", this.traceObjectId, this.segmentStartOffset.get(), newStartOffset);
-            this.segmentStartOffset.set(newStartOffset);
+            log.info("{}: Reconciling start offset from {} to {}.", this.traceObjectId, this.processedUpToOffset.getAndSet(newStartOffset), newStartOffset);
+            log.info("{}: Reconciling last truncation offset from {} to {}.", this.traceObjectId, this.lastTruncationOffset.getAndSet(newStartOffset), newStartOffset);
```

Comment:
maybe we can keep the sets outside of the log lines.

Reply:
Done.